client/src/Components/ResultsTable.js

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import { useTable } from "react-table";
+const [results,setAllResults]=useState();
```

Comment:
the name and function should be the same, like this
const [results, setResults],useState()